packages/webpack-cli/lib/bootstrap.js

```diff
@@ -39,6 +40,11 @@ const runCLI = async (cliArgs) => {
         if (parsedArgs.unknownArgs.length > 0) {
             parsedArgs.unknownArgs.forEach(async (unknown) => {
                 logger.error(`Unknown argument: ${unknown}`);
+                const strippedFlag = unknown.substr(2);
+                const { name: suggestion } = core.find((flag) => leven(strippedFlag, flag.name) < 3);
+                if (suggestion) {
+                    logger.raw(`Did you mean --${suggestion}?`);
+                }
```

Comment:
Can we implement it for commands too? 

`webpack server` -> `Did you mean `serve`?`

Reply:
I don't think that would be a great idea since we allow specifying entry-files directly (`webpack-cli ./index.js`).